src/elements/LayoutAuth.tsx

```diff
@@ -14,6 +14,64 @@ export type AuthOutletContextType = {
   setSidebar: React.Dispatch<React.SetStateAction<ReactNode>>
 }
 
+const images = [
+  '/assets/illustrations/1.png',
+  '/assets/illustrations/2.png',
+  '/assets/illustrations/3.png',
+  '/assets/illustrations/4.png',
+  '/assets/illustrations/5.png',
+  '/assets/illustrations/6.png',
+  '/assets/illustrations/7.png',
+  '/assets/illustrations/8.png',
+  '/assets/illustrations/1.png',
+  '/assets/illustrations/2.png',
+  '/assets/illustrations/3.png',
+  '/assets/illustrations/4.png',
+  '/assets/illustrations/5.png',
+  '/assets/illustrations/6.png',
+  '/assets/illustrations/7.png',
+  '/assets/illustrations/8.png',
+]
+
+//ToDo: Move the text to the i18n file, translate them and call the t() function
+const titles = [
+  'Simplified Setup:',
+  'Flexible Plans:',
+  'Cost Savings:',
+  'Universal Accessibility:',
+  'Trustworthy Technology:',
+  'Empower Participation:',
+  'Seamless Integration:',
+  "Relax, We've Got You Covered:",
+  'Streamline Operations:',
+  'Stay Informed:',
+  'Boost Participation:',
+  'Control Costs:',
+  'Enhanced Flexibility:',
+  'Build Trust:',
+  'Join hundreds of organizations:',
+  'Let us handle the complexity:',
+]
+
+const texts = [
+  'Launch secure and verifiable elections in just a few clicks.',
+  'Adaptable features that scale with your organization.',
+  'Save up to 10x compared to traditional voting methods.',
+  'Organize votes accessible on any device—mobile, tablet, or desktop.',
+  'Advanced encryption ensures your elections are secure and private.',
+  'Foster higher voter turnout with user-friendly tools.',
+  'Easily incorporate your branding and existing workflows.',
+  'Your voting processes are safe, secure, and ready to go.',
+  'Manage multiple elections effortlessly from one dashboard.',
+  'Real-time updates and insights at your fingertips.',
+  'Increase voter engagement with intuitive voting flows.',
+  'Reduce operational expenses for you and your voters.',
+  'Explore advanced options like anonymous voting or weighted votes.',
+  'Transparent and verifiable elections foster confidence in your results.',
+  'Embrace the future of voting. Start today!',
+  'Focus on what matters most.',
+]
```

Comment:
Do the 2do 🙃 

You can move everything to a hook:
```ts
const useLayoutTranslations = () => {
  const { t } = useTranslation()
  const titles = [
    t('whatever.title.1', { defaultValue: 'Simplified Setup:' ),
    // [...]
  ]
  const texts = [
    t('whatever.text.1', { defaultValue: 'Launch secure and verifiable elections in just a few clicks.' }),
    // [...]
  ]
  return {titles, texts}
}
``` 

And then use that hook in the LayoutAuth component:

```ts
const { titles, texts } = useLayoutTranslations()

// the rest of the code remains the same
```